src/bzflag/BackgroundRenderer.cxx

```diff
@@ -24,6 +24,7 @@
 #include "TextureMatrix.h"
 #include "ParseColor.h"
 #include "BZDBCache.h"
+#include "VBO_Handler.h"
```

Comment:
since all you did was remove code, I don't think this include is needed.

Reply:
OK